docs/dabp/release-notes/index.md

```diff
@@ -0,0 +1,11 @@
+# Release notes
+
+This section contains release notes for Onegini DABP.
+
+In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [Onegini Support](https://support.onegini.com).
+
+## Release date: 2021-04-13
+* Introduced log events via oneex
```

Comment:
OneEx is an internal project. Don't think users should know about it

Reply:
fixed